MediLink/.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+repo_token: 0e0dtfi4LDukeXh40552ON6pzrlTlVBcx
```

Comment:
Delete file and make this as environment variable in travis instead

Reply:
done